iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -874,14 +874,45 @@ public static Network copy(Network network, NetworkFactory networkFactory) {
     }
 
     public static Network copy(Network network, NetworkFactory networkFactory, ExecutorService executor) {
+        return copy(network, networkFactory, executor, TreeDataFormat.XML);
+    }
+
+    /**
+     * Deep copy of the network using the specified converter
+     *
+     * @param network the network to copy
+     * @param format the converter to use to export/import the network
+     * @return the copy of the network
+     */
+    public static Network copy(Network network, TreeDataFormat format) {
+        return copy(network, NetworkFactory.findDefault(), format);
+    }
+
+    /**
+     * Deep copy of the network using the specified converter
+     *
+     * @param network        the network to copy
+     * @param networkFactory the network factory to use for the copy
+     * @param format the converter to use to export/import the network
+     * @return the copy of the network
+     */
+    public static Network copy(Network network, NetworkFactory networkFactory, TreeDataFormat format) {
+        return copy(network, networkFactory, ForkJoinPool.commonPool(), format);
+    }
+
+    public static Network copy(Network network, NetworkFactory networkFactory, ExecutorService executor, TreeDataFormat format) {
         Objects.requireNonNull(network);
         Objects.requireNonNull(networkFactory);
         Objects.requireNonNull(executor);
+        ExportOptions exportOptions = new ExportOptions();
+        exportOptions.setFormat(format);
+        ImportOptions importOptions = new ImportOptions();
+        importOptions.setFormat(format);
```

Comment:
no need to define the options here, you can define them later, only when needed. Plus you can even create them inline,  for instance for export:
```java
write(network, new ExportOptions().setFormat(format), pos);
```